Queueomatic/Shared/DTOs/LoginDto.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Queueomatic.Shared.DTOs;
```

Comment:
Oh, no that is not. `null!` says that the variable can not be set as null.

Reply:
That is not true. By initializing the value to null you set its base value to null. The exclamation mark only supresses the warnings the compiler gives you. This means that the value WILL be null prior to setting its value, but you won't see the warning about the fact that it is null before that.

I would say we remove it as a string's base value is null anyway.

Reply:
As you say the base value of string is null but by setting null-forgiving operator in variable declaration says the compiler that the value will not be null when we initializing the object, if so then it throws a NullReferenceException which we can catch. Otherwise we say that it may be null and it's fine, which it isn't.

Do you still want that we go without?

Reply:
If we include ```!```, we will not receive a highlighted warning when we are using that object. This could lead to some hard to find bugs as the compiler won't tell us what the issue may be. Such errors only show during runtime. 

I still stand by my previous comment. Remove the ```null!``` from every model that has it.

Reply:
Yeah sure. I'll fix it.